### PR TITLE
Add basic meta tags and logo.

### PR DESCRIPTION
**Pull request by zachlatta**

### CODE REVIEW

**Comment by zachlatta** [comment]
Going to go ahead and merge this after having read through it again because @MaxWofford is traveling.

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -3,7 +3,13 @@
   %head
     %meta{charset: 'utf-8'}
     %meta{content: 'width=device-width, initial-scale=1.0', name: 'viewport'}
+
+    %meta{name: 'title', content: 'hackEDU'}
+    %meta{name: 'description', content: 'The international movement of free student-led coding clubs.'}
+    %meta{name: 'og:image', content: "#{ image_path('logo.png') }"}
+
     %title= full_title(yield(:title))
+
```
Comment (MaxWofford) [inline app/views/layouts/application.html.haml]: Is there a reason we decided to have newlines to break up the different sections of the head?

Comment (zachlatta) [inline app/views/layouts/application.html.haml]: Nope, totally arbitrary. Seemed easier to read having the meta tags separated.

Perhaps a better solution would be to move the meta tags to a partial and not have the extra newlines?

Comment (MaxWofford) [inline app/views/layouts/application.html.haml]: We don't have a style-guide laid out for this, so it might be best to just leave as-is for now.

Comment (zachlatta) [inline app/views/layouts/application.html.haml]: Alright, sounds good. I do think moving it to a partial would be cleaner though.

Comment (zachlatta) [inline app/views/layouts/application.html.haml]: Especially considering we have the navbar and flashes in a partial.

Comment (MaxWofford) [inline app/views/layouts/application.html.haml]: Okay. Should we create an issue for it?

Comment (zachlatta) [inline app/views/layouts/application.html.haml]: Just created #166 to track this.

On Wed, Sep 9, 2015 at 5:56 PM, Max Wofford <notifications@github.com>
wrote:

> In app/views/layouts/application.html.haml
> <https://github.com/hackedu/website/pull/157#discussion_r39115056>:
>
> >      %title= full_title(yield(:title))
> > +
>
> Okay. Should we create an issue for it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/website/pull/157/files#r39115056>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us

**Pull request merged**